src/WebJobs.Script.WebHost/WebScriptHostManager.cs

```diff
@@ -169,7 +171,23 @@ public HttpRouteCollection Routes
                 };
                 using (logger.BeginScope(scopeState))
                 {
+                    var stopWatch = new Stopwatch();
+                    stopWatch.Start();
                     await Instance.CallAsync(function.Name, arguments, cancellationToken);
+                    stopWatch.Stop();
+
+                    if (request.IsColdStartRequest())
+                    {
+                        var message = new JObject
+                        {
+                            { "requestId", request.GetRequestId() },
+                            { "functionDuration", stopWatch.ElapsedMilliseconds },
+                            { "language", function.Metadata.ScriptType.ToString() }
```

Comment:
This language will be one of the ScriptType values: https://github.com/Azure/azure-functions-host/blob/v1.x/src/WebJobs.Script/Description/ScriptType.cs. This will allow distinction for example between CSharp (CSX) and DotNetAssembly (precompiled)